src/analysis/retail/paladin/protection/modules/core/ProtPaladinT304P.tsx

```diff
@@ -0,0 +1,140 @@
+import { formatPercentage } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import TALENTS from 'common/TALENTS/paladin';
+import HIT_TYPES from 'game/HIT_TYPES';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, {
+  ApplyBuffEvent,
+  CastEvent,
+  DamageEvent,
+  RefreshBuffEvent,
+} from 'parser/core/Events';
+import { plotOneVariableBinomChart } from 'parser/shared/modules/helpers/Probability';
+import BoringSpellValue from 'parser/ui/BoringSpellValue';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import { TIERS } from '../../../../../../game/TIERS';
+import { gcJudgmentCrit } from '../CastLinkNormalizer';
+import Abilities from '../Abilities';
+
+const JUDGM_PROC_CHANCE = 0.5;
+
+class ProtPaladinT304P extends Analyzer {
+  private gcJudgmentCrits: number = 0;
```

Comment:
you don't really need to make this private, just make the variable public and get rid of the getter.